projects/ui/src/temp-overrides.scss

```
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2016-2024 Broadcom. All Rights Reserved.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+@use '@cds/core/tokens/tokens.scss';
+[cds-theme] {
+  --cds-alias-object-container-backdrop-background: hsla(0, 0%, 100%, 0.6);
+  --cds-alias-object-interaction-background-hover: #{tokens.$cds-global-color-construction-50};
+  --cds-alias-object-interaction-background-active: #{tokens.$cds-global-color-construction-100};
+  --cds-alias-object-interaction-background-selected: #{tokens.$cds-global-color-blue-75};
+  --cds-alias-object-interaction-background-selected-hover: #{tokens.$cds-global-color-construction-100};
+  --cds-alias-object-interaction-background-selected-active: #{tokens.$cds-global-color-construction-200};
+}
+
+[cds-theme~='dark'] {
+  --cds-alias-object-container-backdrop-background: #{tokens.$cds-alias-object-opacity-300};
+  --cds-alias-object-interaction-background-hover: #{tokens.$cds-global-color-construction-800};
+  --cds-alias-object-interaction-background-active: #{tokens.$cds-global-color-construction-600};
+  --cds-alias-object-interaction-background-selected: #{tokens.$cds-global-color-blue-900};
+  --cds-alias-object-interaction-background-selected-active: #{tokens.$cds-global-color-blue-600};
+  --cds-alias-object-interaction-background-selected-hover: #{tokens.$cds-global-color-blue-700};
```

Based on figma `selected-active` and `selected-hover` should have `construction-600` and `construction-700` respectively.

```suggestion
  --cds-alias-object-interaction-background-selected-active: #{tokens.$cds-global-color-construction-600};
  --cds-alias-object-interaction-background-selected-hover: #{tokens.$cds-global-color-construction-700};
```